docs/reference/transform/overview.asciidoc

```diff
@@ -62,3 +62,20 @@ image::images/pivot-preview.jpg["Example of a {transform} pivot in {kib}"]
 IMPORTANT: The {transform} leaves your source index intact. It
 creates a new index that is dedicated to the transformed data.
 
+
+[[transform-performance]]
+==== Performance considerations
+
+As mentioned earlier, {transforms} perform search aggregations on the source 
+indices then index the results into the destination index. It means a 
```

Comment:
Maybe it's just me, but starting a sentence with "it" always makes me a bit confused about what "it" is:
```suggestion
indices then index the results into the destination index. Therefore, a 
```